packages/apps/Makefile

```
@@ -1,14 +1,20 @@
 OUT=../../_out/repos/apps
-TMP=../../_out/repos/apps/historical
+TMP := $(shell mktemp -d)
 
 repo:
 	rm -rf "$(OUT)"
 	mkdir -p "$(OUT)"
-	awk '$$3 != "HEAD" {print "mkdir -p $(TMP)/" $$1 "-" $$2}' versions_map | sh -ex
-	awk '$$3 != "HEAD" {print "git archive " $$3 " " $$1 " | tar -xf- --strip-components=1 -C $(TMP)/" $$1 "-" $$2 }' versions_map | sh -ex
-	helm package -d "$(OUT)" $$(find . $(TMP) -mindepth 2 -maxdepth 2 -name Chart.yaml | awk 'sub("/Chart.yaml", "")' | sort -V)
-	cd "$(OUT)" && helm repo index . --url http://cozystack.cozy-system.svc/repos/apps
-	rm -rf "$(TMP)"
+	cd .. && cat apps/versions_map2 | while read package version commit ; \
```

This should be updated

```suggestion
	cd .. && cat apps/versions_map | while read package version commit ; \
```

Please consider moving this logic outside of the Makefile into separate script.

Alternative implementation can be found here:
- https://github.com/cozystack/cozystack/pull/943

We need to make decision which logic will be better and more safe

---

Please consider moving this logic outside of the Makefile into separate script.

Alternative implementation can be found here:
- https://github.com/cozystack/cozystack/pull/943

We need to make decision which logic will be better and more safe

---

Fixed in hack/package_repo.sh